aspnetcore/host-and-deploy/iis/modules.md

```diff
@@ -99,13 +99,13 @@ For more information on disabling modules with configuration settings, follow th
 
 If opting to remove a module with a setting in *web.config*, unlock the module and unlock the `<modules>` section of *web.config* first:
 
-1. Unlock the module at the server level. Select the IIS server in the IIS Manager **Connections** sidebar. Open the **Modules** in the **IIS** area. Select the module in the list. In the **Actions** sidebar on the right, select **Unlock**. Unlock as many modules as you plan to remove from *web.config* later.
+1. Unlock the module at the server level. Select the IIS server in the IIS Manager **Connections** sidebar. Open the **Modules** in the **IIS** area. Select the module in the list. In the **Actions** sidebar on the right, select **Unlock**. If the action entry for the module appears as **Lock**, the module is already unlocked, and no action is required. Unlock as many modules as you plan to remove from *web.config* later.
```

Comment:
I don't believe this is correct. You need to unlock the modules sections itself rather than unlocking individual modules.

Reply:
This came about from direct testing. This goes back quite a while in the history of the topic ... to when it only pertained to using IIS and I performed the testing when this section for IIS was added.

Note too that for IIS Express, I found the same thing: I had to unlock individual modules to make it work.

Perhaps, I need to retest and confirm. That's fine. I can do that.

Reply:
Yeah ... it goes back to *ancient times* (2 years ago) on https://github.com/aspnet/Docs/pull/2609

Reply:
😕

@bariscaglar Does this seem right to you?

Reply:
Note btw that this is at **_the IIS server level_**. I'm going to run a test now to see what happens if I lock a module and unlock the section ... see if it :booms: on me.